crates/rspack_core/src/normal_module.rs

```diff
@@ -398,6 +402,7 @@ impl Module for NormalModule {
         code_generation_dependencies: &mut code_generation_dependencies,
         build_info: &mut build_info,
         build_meta: &mut build_meta,
+        devtool: &devtool,
```

Comment:
should have this in ParseResult

Reply:
This `devtool` setting will be used in the parse function, which is not asynchronous.